src/HostManager.cpp

```diff
@@ -214,8 +214,8 @@ void HostManager::postInterHostEvent( const Host * pHost, const TEvent & event )
     allValidHosts = afterSendingHost;
     allValidHosts.append( beforeSendingHost );
 
-    for( int j = 0, total = allValidHosts.size(); j < total; ++j ) {
-        hostList.at( allValidHosts.at( j ) )->raiseEvent( event );
+    for(int allValidHost : allValidHosts) {
```

Comment:
The use of `total` and the pre-increment is the result of allowing/using the Qt Creator context menu's "Refactor" -> "optimise loop" on the `for(...) {...}` statement.